lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -164,10 +171,17 @@ async def _sync_subscribers(self) -> bool:
                 self._loop,
             )
         except grpc.RpcError as err:
-            logging.error(
-                "Sync subscribers request error! [%s] %s", err.code(),
-                err.details(),
-            )
+            if err.code() == grpc.StatusCode.INTERNAL:
+                logging.info(
+                    'Internal Error in subscriberdb-Orc8r communication. '
+                    'If AGW and Orc8r are deployed at a commit hash, '
+                    'upgrade AGW and Orc8r to latest commit hash.'
+                )
+            else:
+                logging.error(
+                    "Sync subscribers request error! [%s] %s", err.code(),
+                    err.details(),
+                )
```

Comment:
Since it's the same logic in both cases (and almost the same messages), I think you can extract a function to handle the logging.